app/javascript/components/components/ProjectComponent.vue

```diff
@@ -104,7 +106,7 @@
           </b-tab>
         </b-tabs>
       </b-col>
-      <b-col v-if="effective_permissions" md="2">
+      <b-col v-if="effective_permissions" md="4">
```

Comment:
This makes the center column feel really cramped on small displays. Would it be possible to go with md-3 instead of md-4 here and md-9 instead of md-8 below?

Reply:
ah yes, I changed it before removing the ability to add reviews (which took up a lot more space), will change it back

Reply:
https://github.com/mitre/vulcan/pull/331/commits/c2b483b47ad7cdd1ced0d66b9e1fc70d4eb21d15